pkg/twittermeow/polling.go

```diff
@@ -90,3 +89,8 @@ func (pc *PollingClient) stopPolling() {
 func (pc *PollingClient) SetActiveConversation(conversationID string) {
 	pc.activeConversationID = conversationID
 }
+
+func (pc *PollingClient) pollConversation(conversationID string) {
+	pc.includeConversationID = true
+	pc.startPolling(context.Background())
+}
```

Comment:
Instead of restarting polling every time, there should just be a channel for short-circuiting the poll ticker

With `pc.shortCircuit` set to a `make(chan struct{}, 1)`, something like this in the loop:

```go
select {
case <-pc.shortCircuit.GetChan():
  tick.Reset()
  fallthrough
case <-tick.C:
  // existing poll code
```

and then to wake it up (with safety to prevent queuing multiple polls for no reason):

```go
select {
case pc.shortCircuit <- struct{}{}:
default:
}
```